2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,14 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+let recipe = {
+  title: "Mole",
+  servings: 2,
+  ingredients: ["cinnamon", "cumin", "cocoa"],
+};
+
+console.log(recipe.title);
+console.log(`Serves: ${recipe.servings}`);
+console.log("Ingredients:");
+recipe.ingredients.forEach((ingredient) => console.log(ingredient));
```

Comment:
good job, but here we wanted you to have at least 5 receipts and show each of those receipts in a separate line, can you change your solution? :)